addons/mail/models/mail_thread.py

```diff
@@ -2024,7 +2024,7 @@ def _partner_find_from_emails(self, records_emails, avoid_alias=True, ban_emails
         # fetch information used to find existing partners, beware portal/public who
         # cannot read followers
         followers = self.sudo().message_partner_ids if 'message_partner_ids' in self else self.env['res.partner']
-        alias_emails = self.env['mail.alias.domain'].sudo()._find_aliases(emails_key_all) if avoid_alias else []
+        alias_emails = self.env['mail.alias.domain'].sudo()._find_aliases_and_localparts(emails_key_all) if avoid_alias else []
```

Comment:
Means any override won't apply here anymore :/
Can't we implement it on top of `_find_aliases`? Just doing 2 searches